packages/web-components/fast-element/docs/guide/observables-and-state.md

```diff
@@ -143,9 +143,37 @@ notifier.subscribe(handler, 'firstName')
 notifier.unsubscribe(handler, 'lastName');
 ```
 
+## Observing Arrays
+
+So far, we've only seen how to observe properties on objects, but it's also possible to observe arrays for changes. Given an instance of an array, it can be observed like this:
+
+**Example: Observing an Array**
+
+```ts
+const arr = [];
+const notifier = Observable.getNotifier(arr);
+const handler = {
+  handleChange(source: any, splices: Splice[]) {
```

Comment:
It doesn't appear that `Splice` is in the fast-element API docs because it isn't exported from the main `index.ts`. It's marked as public though, so it probably should be. Can we export that as part of this change as well?

Reply:
Good catch! I noticed another api that needed to be exported too and then that prompted me to think through adding some more docs. I'll get those handled and push the update shortly.

Reply:
Ok, got it taken care of.